packages/styletron-react/src/types.js

```diff
@@ -43,6 +43,16 @@ export type Styletron = {
   },
 };
 
+export type StyletronExtension = {
```

Comment:
This object seems pretty much identical to the `Styletron` object. Given there's just a few additional properties, I'm thinking it might make sense to add them as an optional field to the styletron object (similar to `debug` and `stackIndex`/`stackInfo`). Then `createStyledElementComponent` could still only take one argument and therefore remain composable.

Reply:
Totally 👍

Reply:
This type is unused now, right? It seems it isn't actually used anywhere.